rm_hw:
  bus:
    - can0
  loop_frequency: 1000
  cycle_time_error_threshold: 0.001
  # Configurations of the actuators
  actuators:
    left_wheel_actuator:
      bus: can0
      id: 0x002
      type: cheetah
      lp_cutoff_frequency: 100
    right_wheel_actuator:
      bus: can0
      id: 0x001
      type: cheetah
      lp_cutoff_frequency: 100
  imus:
    base_imu:
      frame_id: base_link
      bus: can0
      id: 0x300
      orientation_covariance_diagonal: [ 0.0012, 0.0012, 0.0012 ]
      angular_velocity_covariance: [ 0.0004, 0.0004, 0.0004 ]
      linear_acceleration_covariance: [ 0.01, 0.01, 0.01 ]
